### DISCUSSION TITLE
add physics interpolation to default built in , in godot4

### DISCUSSION THREAD

**Discussion by djmaesen** [godotengine/godot-proposals#4573] [Engine Core]
i would like to see physics interpolation integrated by default in godot4

**Comment by Zireael07** [comment]
Should already be in? #52846 and follow up are merged

**Reply by Calinou** [reply]
It's only in Godot 3.x so far, not 4.0 yet.

The physics interpolation support *will* be ported to Godot 4, but I don't know when exactly it will be done.

**Reply by djmaesen** [reply]
ok good news then
i can wait , now i must use Vsync to avoid stutter
